agent/engine/docker_image_manager.go

```diff
@@ -84,6 +84,12 @@ func (imageManager *dockerImageManager) AddAllImageStates(imageStates []*image.I
 	}
 }
 
+func (imageManager *dockerImageManager) GetCountImageStates() int {
```

Comment:
Instead make this function used only in test, what about add a function return all the imagestates, eg: "AddImageStates" which can also be used for other purpose?